source/includes/usage-examples/run-command.c

```diff
@@ -0,0 +1,157 @@
+#include <bson/bson.h>
+#include <mongoc/mongoc.h>
+#include <stdio.h>
+
+int
+main (int argc, char *argv[])
+{
+    mongoc_client_t *client;
+    mongoc_collection_t *collection;
+    char *str;
+    bson_error_t error;
+
+    mongoc_init ();
+
+    client =
+        mongoc_client_new ("<connection string URI>");
+    collection = mongoc_client_get_collection (client, "<database name>", "collection name");
+
+    {
+        // start-execute-command
+        bson_t *command, reply;
+        bson_error_t error;
+        bool retval;
+    
+        command = BCON_NEW ("hello", BCON_INT32 (1));
+        retval = mongoc_client_command_simple (client, "admin", command, NULL, &reply, &error);
+    
+        if (!retval) {
+            fprintf (stderr, "Failed to run command: %s\n", error.message);
+        } else {
+         char *str = bson_as_canonical_extended_json (&reply, NULL);
+            printf ("%s\n", str);
+            bson_free (str);
+        }
+    
+        bson_destroy (command);
+        bson_destroy (&reply);
+        // end-execute-command
+    }
+
+    {
+        // start-execute-command-with-options
+        bson_t *command;
+        bson_t *opts;
+        bson_t reply;
+        bson_error_t error;
+        bool retval;
+
+        command = BCON_NEW(
+            "grantRolesToUser", BCON_UTF8("user011"),
+            "roles", "[", BCON_UTF8("readWrite"), "]"
+        );
+
+        opts = BCON_NEW(
+            "writeConcern", "{",
+                "w", BCON_UTF8("majority"),
+            "}"
+        );
```

Comment:
Suggest using `mongoc_write_concern_t` to add write concern (as is done below).

```suggestion

        // Includes write concern "majority" in command options
        mongoc_write_concern_t* write_concern = mongoc_write_concern_new ();
        mongoc_write_concern_set_w (write_concern, MONGOC_WRITE_CONCERN_W_MAJORITY);
        bson_t *opts = bson_new ();
        mongoc_write_concern_append (write_concern, opts);
```